### PR TITLE
[RAPTOR-7352] Fix keras vizai pipeline

### PR DESCRIPTION
**Pull request by drenfr01**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

This template was broken because it:
 1) imported torch for no reason (which isn't in the dependencies list)
 2) tried to download weights at fit time (which won't work because it's our docker container has no internet access) 
 3) used an old version of MobileNet which is out of sync with core DataRobot (v2 vs. v3)

## Rationale

We need working templates, this came up because a CFDS tried to use this template and couldn't

### CODE REVIEW

File: task_templates/3_pipelines/15_python3_keras_vizai_joblib/model_utils.py [hunk 1]
```diff
@@ -158,7 +151,10 @@ def get_image_augmentation_gen(X_data, y_data, bs, seed) -> Iterator[tuple]:
 
 def get_pretrained_base_model() -> Model:
     """ A base pretrained model to build on top of """
-    pretrained_model = MobileNetV2(include_top=False, input_shape=IMG_SHAPE, classes=NUM_CLASSES)
+    weights_file = "mobilenetv3_large.h5"
+    pretrained_model = MobileNetV3Large(
```
Comment (jasonjmcghee) [inline task_templates/3_pipelines/15_python3_keras_vizai_joblib/model_utils.py:155]: Can we use `Small` instead? Should _dramatically_ improve runtime

File: task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml [hunk 2]
```diff
@@ -19,6 +19,7 @@ typeSchema:
       condition: IN # can be one of IN or NOT_IN
       value: # can be one or multiple of NUM, TXT, IMG, DATE, CAT, DATE_DURATION, COUNT_DICT, GEO
         - IMG
+        - NUM
```
Comment (scottp-ml) [inline task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml:22]: We shouldn't make this change.  @drenfr01 lets talk tomorrow

**Review by scottp-ml — CHANGES_REQUESTED** [review]

File: task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml (hunk 2 above)
Comment (drenfr01) [reply task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml:22]: kk!

Comment (jasonjmcghee) [reply task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml:22]: FWIW I told @drenfr01 about this change - I could not get it to succeed until I made this change.

Comment (scottp-ml) [reply task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml:22]: From what I had seen the model was after another task that converted from img -> num.  I did a bunch of testing on the same dataset and the type schema was working correctly for me.

Comment (jasonjmcghee) [reply task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml:22]: If it's working it's working. Looks like all the models on that project got deleted, so I can't find the one that had that error that prompted us to make the change, but AFAIK there were no other tasks.

Comment (jasonjmcghee) [reply task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml:22]: Yeah I can't reproduce the problem anymore. I will take another crack at it another time, but if it works by uploading to DataRobot and running it (not just running with drum locally - does this ensure no network?) and it succeeds, I think I'm good with it

Comment (scottp-ml) [reply task_templates/3_pipelines/15_python3_keras_vizai_joblib/model-metadata.yaml:22]: An improvement we need to make is how we handle projects like that one in drum locally, but it is way beyond the scope of this ticket.

**Comment by scottp-ml** [comment]
jarvis test functional_training_model_templates

**Review by scottp-ml — APPROVED** [review]

**Review by jasonjmcghee — APPROVED** [review]

**Pull request merged**